src/GerryChain.jl

```diff
@@ -23,13 +23,18 @@ satisfy_constraint,
 
 # recom
 sample_subgraph, build_mst, add_edge_to_mst!, remove_edge_from_mst!,
-traverse_mst, get_balanced_proposal, get_valid_proposal, update!,
-recom_chain
+traverse_mst, get_balanced_proposal, get_valid_proposal, update_partition!,
+recom_chain,
+
+# measures
+AbstractMeasure, Election, RacePopulation,
```

Comment:
Lol, the first thing that comes to mind is [a measure in the analysis sense.](https://en.wikipedia.org/wiki/Measure_(mathematics)